bin/release

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+
+set -eu
+
+DEFAULT_KEYSTORE_PATH="$PWD/certs/metabase_keystore.jks"
+DEFAULT_KEY_PEM_PATH="$PWD/certs/key.pem"
+DEFAULT_CONFIG_JSON_PATH="$PWD/config.json"
+
+if [ $# -lt 1 ]; then
+    echo "usage: $0 X.Y.Z [BRANCH]"
+    exit 1
+fi
+VERSION=$1
+
+if [ $# -lt 2 ]; then
+    BRANCH="release-$VERSION"
+else
+    BRANCH=$2
+fi
+
+# check that docker is running
+docker ps > /dev/null
+
+# ensure DockerHub credentials are configured
+if [ -z ${DOCKERHUB_EMAIL+x} ] || [ -z ${DOCKERHUB_USERNAME+x} ] || [ -z ${DOCKERHUB_PASSWORD+x} ]; then
+    echo "Ensure DOCKERHUB_EMAIL, DOCKERHUB_USERNAME, and DOCKERHUB_PASSWORD are set.";
+    exit 1
+fi
+
+# ensure AWS is configured
+if [ -z ${AWS_DEFAULT_PROFILE+x} ]; then
+    echo "Setting AWS Profile to 'default'.";
+    AWS_DEFAULT_PROFILE=default
+fi
+
+# ensure we have access to the keystore
+if [ -f "$DEFAULT_KEYSTORE_PATH" ]; then
+    KEYSTORE_PATH="$DEFAULT_KEYSTORE_PATH"
+else
+    echo "Enter the path to JAR signing keystore ($DEFAULT_KEYSTORE_PATH by default)"
+    read KEYSTORE_PATH
+    [ -f "$KEYSTORE_PATH" ] || exit 1
+fi
+
+# # ensure we've configured the Mac signing key
```

Comment:
If you're going to check in a bunch of commented-out code I would say at least add a comment explaining why it's commented out